lib/index.js

```diff
@@ -116,7 +117,8 @@ const setup = ({ customTextFormattings, parseFigureProps, customCaption }: Setup
       // Check props first without rendering,
       // no need to go through with rendering if we know nothing has changed.
       const propsHasUpdated = this.props.selections !== nextProps.selections ||
-        this.props.contentEditable !== nextProps.contentEditable;
+        this.props.contentEditable !== nextProps.contentEditable ||
+        !deepEqual(this.props.items, nextProps.items, { strict: true });
```

Comment:
@ryanscottaudio Hey, did this make a performance difference? Realizing now it might do that in the example, but where we use this in our editor we're using immutable data, so just checking `this.props.items !== nextProps.items` used to be enough before at least.